server/app/views/components/Icons.java

```diff
@@ -85,6 +85,15 @@ public enum Icons {
           + " 2.792Q14 2.271 14.719 2.271Q15.438 2.271 15.958 2.792L17.208 4.042Q17.708 4.542"
           + " 17.708 5.281Q17.708 6.021 17.208 6.521L6.229 17.5ZM15.958 5.271 14.729 4.042ZM13.5"
           + " 7.729 12.896 7.104 12.271 6.5V6.479L13.5 7.729Z"),
+  VIEW(
```

Comment:
**suggestion (blocking):** please add a comment either a description of the icon or linking to a visual reference of the icon